ansible/roles/zookeeper/tasks/deploy.yml

```diff
@@ -17,13 +17,6 @@
 ---
 # This role will install Kafka with Zookeeper in group 'kafka' in the environment inventory
 
-- name: "pull the zookeeper:{{ zookeeper.version }} image"
```

Comment:
Looks like intuitive coding the same as manual operation.

Reply:
The retry here is meaningless, since errors happened here are fatal for deployment.
e.g. image name/tag spelled wrong, wrong password for docker repo, network timeout, etc.

All we should do is to fix them and re-deploy, instead of retrying.

Reply:
The retry here is meaningless, since errors happened here are fatal for deployment.
e.g. image name/tag spelled wrong, wrong password for docker repo, network timeout, etc.

All we should do is to fix them and re-deploy, instead of retrying.

Reply:
@rabbah 
Sorry for my bold clicked. ORZ
Not very familiar with the Github buttons.